src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/update/PopulateUpdateSubmission.java

```diff
@@ -0,0 +1,116 @@
+    updateSubmission.setDueDiligence(submissionDueDiligence);
+
+  }
+
+  public DueDiligence populateByDueDiligenceDto(
+      DueDiligence dueDiligence, DueDiligenceDto dueDiligenceDto) {
+
+    dueDiligence.setDateChecked(dueDiligenceDto.getIdentityDate().toString());
+    dueDiligence.setAgentName(dueDiligenceDto.getName());
+    dueDiligence.setDueDiligenceCorrespondenceAddress(dueDiligenceDto.getAddress());
+    dueDiligence.setSupervisoryBody(dueDiligenceDto.getSupervisoryName());
+    dueDiligence.setPartnerName(dueDiligenceDto.getPartnerName());
+    dueDiligence.setEmail(dueDiligenceDto.getEmail());
+    dueDiligence.setAgentAssuranceCode(dueDiligenceDto.getAgentCode());
+    dueDiligence.setDiligence(dueDiligenceDto.getDiligence());
+
+    return dueDiligence;
+  }
+
+  public DueDiligence populateByOEDueDiligenceDto(
+      DueDiligence dueDiligence, OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto) {
+
+    dueDiligence.setDateChecked(overseasEntityDueDiligenceDto.getIdentityDate().toString());
+    dueDiligence.setAgentName(overseasEntityDueDiligenceDto.getName());
+    dueDiligence.setDueDiligenceCorrespondenceAddress(overseasEntityDueDiligenceDto.getAddress());
+    dueDiligence.setSupervisoryBody(overseasEntityDueDiligenceDto.getSupervisoryName());
+    dueDiligence.setPartnerName(overseasEntityDueDiligenceDto.getPartnerName());
+    dueDiligence.setEmail(overseasEntityDueDiligenceDto.getEmail());
+    dueDiligence.setAmlRegistrationNumber(overseasEntityDueDiligenceDto.getAmlNumber());
+
+    return dueDiligence;
+  }
+
+  public void populateFilingForDate(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    FilingForDate filingForDate = new FilingForDate();
+    var filingDate = overseasEntitySubmissionDto.getUpdate().getFilingDate();
+    filingForDate.setDay((String.valueOf(filingDate.getDayOfMonth())));
+    filingForDate.setMonth((String.valueOf(filingDate.getMonth())));
+    filingForDate.setYear((String.valueOf(filingDate.getYear())));
+    updateSubmission.setFilingForDate(filingForDate);
+  }
+
+  public void populatePresenter(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    Presenter presenter = new Presenter();
+    if (overseasEntitySubmissionDto.getPresenter() != null) {
+      presenter.setEmail(overseasEntitySubmissionDto.getPresenter().getEmail());
+      presenter.setName(overseasEntitySubmissionDto.getPresenter().getFullName());
+    }
+    updateSubmission.setPresenter(presenter);
+  }
+
+  public UpdateSubmission populate(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    this.overseasEntitySubmissionDto = overseasEntitySubmissionDto;
+    this.updateSubmission = updateSubmission;
+    return populate();
+  }
+
+  public void setOverseasEntitySubmissionDto(
```

Comment:
no need for this to be public. Test coverage is still very low and any getter/setters that are not used / tested are going to show up in that.